sdk/core/core-tracing/.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "plugins": ["@azure/azure-sdk"],
   "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
   "rules": {
-    "@azure/azure-sdk/ts-no-const-enums": "off"
+    "@azure/azure-sdk/ts-no-const-enums": "off",
+    "@azure/azure-sdk/ts-versioning-semver": "warn"
```

Comment:
?

Reply:
eslint enforces the use of alpha/beta not the older preview name.

Reply:
I had to switch back to the -preview naming (which violates this rule now, so it has to be disabled).